### DISCUSSION TITLE
Fetching Volumes instead of Chapters

### DISCUSSION THREAD

**Discussion by nickstu** [manga-download/haruneko#835] [Ideas] [answered]
Hi, first of all a huge thanks to people working on this project.
I have a technichal question. A few days ago I was trying to download something from Zebrack, and I saw that by default I could only retrieve the chapter list, while I was interested in the volume list. After some playing around with the code, I figured that I could just change the Manga id to say "volume" instead of "chapter", and that would immediatly give me the volume list. I'm now wondering, is there a "proper" way to do this inside haruneko, without changing lines of code?
If not, I think it could be added as an option, which is why I'm posting this as "Ideas".

**Comment by MikeZeDev** [comment, marked as answer]
There is a pending pr for zebrack but most website doesnt offer volumes so forget about it.
